editor/src/components/editor/action-types.ts

```diff
@@ -727,6 +727,7 @@ export interface SetProp {
   target: ElementPath
   propertyPath: PropertyPath
   value: JSExpression
+  importsToAdd?: Imports
```

Comment:
Usually we prefer non-optional function parameters, so you can not forget about them accidentally.

Reply:
Absolutely, but I wanted to avoid adding a ton of unrelated changes to the PR because of this. Also, the command constructor functions use the same pattern. I don't have strong opinions on this, I can update the PR if we don't want the optional param here

Reply:
I made `importsToAdd` non-optional in the action type, and defaulted it to an empty imports object https://github.com/concrete-utopia/utopia/pull/5050/commits/71276a4a4eeca3b26443c699cab9ab9e5072feae